contracts/world/sources/character/character.move

```diff
@@ -44,8 +46,7 @@ public struct CharacterCreatedEvent has copy, drop {
     game_character_id: u32,
     tenant: String,
```

Comment:
can replace those by the key if you want 👍 

```suggestion
key: DerivationKey,
```

Reply:
you can send as a struct from the client ?